src/main/java/util/Parser.java

```diff
@@ -0,0 +1,39 @@
+package util;
+
+import java.util.HashMap;
+import java.util.InputMismatchException;
+
+public interface Parser {
+    static String getCommand(String input) {
+        return input.split(" ", 2)[0];
+    }
+
+    static HashMap<String, String> getArgMap(String input) throws InputMismatchException {
+        HashMap<String, String> argMap = new HashMap<>();
+
+        if (input.split(" ").length < 2) {
+            return argMap;
+        }
+
+        String[] argArr = input.split("/");
```

Comment:
Maybe you would like to rename it to a more informative name, e.g. argumentsFromInput (might not be the best name, but I can't really think of other alternatives)